core/src/commitment_service.rs

```diff
@@ -139,8 +148,11 @@ impl AggregateCommitmentService {
         aggregation_data: CommitmentAggregationData,
         ancestors: Vec<u64>,
     ) -> CommitmentSlots {
-        let (block_commitment, rooted_stake) =
-            Self::aggregate_commitment(&ancestors, &aggregation_data.bank);
+        let (block_commitment, rooted_stake) = Self::aggregate_commitment(
+            &ancestors,
+            &aggregation_data.bank,
+            Some(&aggregation_data.node_vote_state),
```

Comment:
Hmm, `node_vote_state` is never None...? Why bother with the Option at all?

Reply:
i forgot to clean up this before marking this pr out of draft...: https://github.com/anza-xyz/agave/pull/1964/commits/e5c5d95398780f27914aaeeaf9d2067d1824da18